memgpt/agent.py

```diff
@@ -931,6 +933,34 @@ def update_state(self) -> AgentState:
             "memory": self.memory.to_dict(),
             "messages": [str(msg.id) for msg in self._messages],  # TODO: move out into AgentState.message_ids
         }
+
+        ## Additional metadata we want to track (that's useful for the client to know)
```

Comment:
would it be ok if we put this in the `update_state` method? then we can have all the agent state management in one place

Reply:
moved out to separate REST routes